test/controllers/static_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class StaticControllerTest < ActionController::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Hmmm… the static controller is only temporary in order to render the template. Therefore, I'd prefer to remove this file.